newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackPurchaseDialog.js

```diff
@@ -130,45 +142,36 @@ const PrivateAssetPackPurchaseDialog = ({
     []
   );
 
-  const checkUserPurchases = React.useCallback(
```

Comment:
simplify logic like on the gametemplate purchase dialog, the purchases are now stored in the context, so we just have to read them